lib/translations/en-US.json

```diff
@@ -50,9 +50,9 @@
         "featuredAppsTitle": "Featured apps",
         "featuredAppsSubtitle": "Ready to try? Subscribe to one of your featured apps to get started!",
         "ctaCard": {
-          "buttonLabel": "Create application",
+          "buttonLabel": "Go to portal",
           "title": "Are you a Developer?",
-          "text": "Create and publish your own applications on {{portalOwner}}'s Marketplace through our developer portal!"
+          "text": "Publish and market your own applications on {{portalOwner}}'s Marketplace through our developer portal!"
```

Comment:
Is this text correct?

Reply:
Yes, this is the text used by the design that precipitated the CTA change.